package.json

```diff
@@ -28,8 +28,8 @@
     "typescript": "^4.1.2"
   },
   "scripts": {
-    "test": "npx hardhat test ./test/unit/*.ts",
-    "test-integration": "npx hardhat --config hardhat.integration-test-config.ts test ./test/integration/*.ts",
+    "test": "npx hardhat test ./test/unit/**/*.ts",
+    "test-integration": "npx hardhat --config hardhat.integration-test-config.ts test ./test/integration/**/*.ts",
```

Comment:
this failed on my machine, are you sure its not `integration/*.ts` ? the integration directory isn't just a single layer deep?

Reply:
integration/**/*.ts is a glob that recursively includes all dirs from integration on. it's a safer bet if/when we add more test dirs. tests fail either way in the same way (using ** or not)